week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -1,6 +1,6 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
+function addNumbers(a ,b ,c) {
```

Comment:
Just a minor detail, but please make sure to have a space after `,` and no space before. Consistency is important.